tls/s2n_resume.c

```diff
@@ -326,7 +324,7 @@ static S2N_RESULT s2n_deserialize_resumption_state(struct s2n_connection *conn,
     uint8_t format = 0;
     RESULT_GUARD_POSIX(s2n_stuffer_read_uint8(from, &format));
 
-    if (format == S2N_SERIALIZED_FORMAT_TLS12_V1 || format == S2N_SERIALIZED_FORMAT_TLS12_V2) {
+    if (format == S2N_SERIALIZED_FORMAT_TLS12_V3) {
```

Comment:
oh, I like that, it's a lot cleaner solution compared to what I had in the issue.